CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Add the possibility to add custom classes to notifications.  
+Setting the `apos-notification--hidden` class will hide the notification, which can be useful when we only care about the event carried by it.
```

Comment:
Work with Bob to document the use case for this one in particular. I'm a little fuzzy on it myself.

Reply:
cc @BoDonkey